javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,36 @@
+const redBox = document.querySelector('ul.marks li:nth-child(1)');
+const blueBox = document.querySelector('ul.marks li:nth-child(2)');
+const greenBox = document.querySelector('ul.marks li:nth-child(3)');
+const positions = [
+    {x: 20, y: 300},
+    {x: 400, y: 300}, 
+    {x: 400, y: 20}
+]
```

Comment:
Great job! Your code is clean and clear. 
The only thing I would suggest to add here is a calculation of the exact positioning of the targets. In the provided code (in `move-element.js` file) we have `function randomizeStartPosition()`, which defines the start position with random x and y integers (from -20 to 20). I just added one more function to my code to calculate the exact position: 
```
function calculateTargetPosition(targetPosition, boxToMove) {
    const boxToMoveTargetLeft = targetPosition.x - parseInt(boxToMove.style.left);
    const boxToMoveTargetTop = targetPosition.y - parseInt(boxToMove.style.top);
    return { x: boxToMoveTargetLeft, y: boxToMoveTargetTop };
}
```
The benefit of adding this function is that you will receive a huge thumbsup when the circle reaches its target :)

Reply:
Wow thanks so much for explaining and the code :) I could not actually figure this out. Will change that part.